database/postgres/pg.test.js

```diff
@@ -0,0 +1,80 @@
+jest.mock('pg')
+const {Client} = require('pg')
+const {startPGDB, closePGDB, createPgAccount, deletePgAccount} = require('./pg')
+
+describe('Test PG DB', ()=>{
+  afterEach(()=>{
```

Comment:
This should be `beforeEach(...)` shoudn't it?

Reply:
It could be `beforeEach` as well but on the first test the mocks would be cleared when there is no need to clear.

Reply:
should be `beforeEach`. As the test suite gets more complex, this will cause issues later on, you can't rely on other ppl to always clear their test.